packages/timeline-state-resolver/src/integrations/tricaster/__tests__/triCasterShortcutStateConverter.spec.ts

```diff
@@ -0,0 +1,286 @@
+import { TriCasterShortcutStateConverter } from '../triCasterShortcutStateConverter'
+
+function setUpShortcutStateConverter() {
+	return new TriCasterShortcutStateConverter(
+		['main', 'v1', 'v2'],
+		['input1', 'input2'],
+		['input1', 'input2', 'sound', 'master'],
+		['a', 'b'],
+		['dsk1', 'dsk2', 'dsk3', 'dsk4'],
+		['mix1', 'mix2']
+	)
+}
+
+describe('TriCasterShortcutStateConverter.getTriCasterStateFromShortcutState', () => {
+	describe('MixEffects', () => {
+		test('sets inputs', () => {
+			const converter = setUpShortcutStateConverter()
+
+			const state = converter.getTriCasterStateFromShortcutState(`<shortcut_states>
+	<shortcut_state name="main_a_row_named_input" value="INPUT7" type="" sender="unknown"/>
+	<shortcut_state name="main_b_row_named_input" value="DDR2" type="" sender="unknown"/>
+</shortcut_states>`)
+
+			expect(state.mixEffects['main'].programInput).toEqual('input7')
+			expect(state.mixEffects['main'].previewInput).toEqual('ddr2')
+		})
+
+		test.skip('sets cut transition', () => {
```

Comment:
I take that the follow about a dozen or so `test.skip(...)`, are tests that is missing some external, yet to be implented, code, for the tests to give an correct result?
If so, could those external parts not be mocked?

I don't fully understand why one would write tests, that is then skipped, if it is intended to stay like this.

Please do enlighten me as too why one would make use of the `test.skip(...)`.

Reply:
> If so, could those external parts not be mocked?

I don't think so. The functionality covered by those test had to be disabled/removed for now. They represent what we would eventually like the API we're integrating with to provide reliably, but it does not seem to be doing so yet, hence having this functionality in place could introduce bugs.

> I don't fully understand why one would write tests, that is then skipped

They were written before coming to conclusion that we can't trust the data.

I removed those test anyway because I didn't feel like maintaining them after the recent types change.